public/components/create_report/create_report.tsx

```diff
@@ -38,14 +38,34 @@ import {
     EuiRadioGroup,
   } from '@elastic/eui';
 import { htmlIdGenerator } from '@elastic/eui/lib/services';
-import moment from 'moment';
+import moment, { Moment } from 'moment';
 import { ReportSettings } from './report_settings/report_settings';
 import { ReportDelivery } from './delivery/delivery';
 import { ReportSchedule } from './schedule/schedule';
 
 const idPrefix = htmlIdGenerator()();
 
-export class CreateReport extends React.Component {
+interface RouterHomeProps {
+  httpClient?: any
+}
+
+interface CreateReportState {
+  reportSettingsRadioIdSelected: string,
+  reportSettingsSetRadioIdSelected: string,
+  reportSettingsDashboard: string,
+  deliveryEmailSubject: string,
+  deliveryEmailBody: string,
+  scheduleRadioFutureDateSelected: boolean,
+  scheduleRadioRecurringSelected: boolean,
+  scheduleUTCOffset: number,
+  scheduleRecurringFrequency: string,
+  scheduleRecurringDailyTime: string, 
```

Comment:
Shouldn't this be number ? Is it ISO string or timestamp ?

Reply:
Looks like this variable isn't even used, removing it altogether